plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -31,6 +31,9 @@ public function __construct() {
 			}
 			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 			add_filter( 'woocommerce_register_post_type_product', array( $this, 'add_product_template' ) );
+
+			add_action( 'current_screen', array( $this, 'set_current_screen_to_block_editor_if_wc_admin' ) );
```

Comment:
I tried moving this to the top of the `enqueue_scripts` function in this class, but it must be getting called after GB's initialization. I think leaving it hooked up to `current_screen` is the best option now rather than fooling around with priorities on an `admin_enqueue_scripts` handler to try and get our code executed before GB's initialization.

Reply:
That makes sense.  Thanks for the explanation!